.github/workflows/build-docker.yaml

```diff
@@ -0,0 +1,60 @@
+name: Build Docker Images
+
+on: [push, pull_request]
```

Comment:
we are currently running the same job twice per PR. this PR ran 4 workflows (2 dulicates). Is that intended?

Reply:
No, this should be on push to main and pull requests not just push.